src/Compilers/Core/Portable/InternalUtilities/ImmutableArrayExtensions.cs

```diff
@@ -47,7 +47,7 @@ internal static ImmutableArray<T> ToImmutableArrayOrEmpty<T>(this ImmutableArray
             return ~low;
         }
 
-        internal static ImmutableArray<TDerived> CastDown<TOriginal, TDerived>(this ImmutableArray<TOriginal> array) where TDerived : class?, TOriginal
+        internal static ImmutableArray<TDerived> CastDown<TOriginal, TDerived>(this ImmutableArray<TOriginal> array) where TDerived : class, TOriginal
```

Comment:
📝 The original appears to be correct here as well. Can you provide more context?

Reply:
This is how it's annotated in the framework.